src/build_platform/local_docker.rs

```diff
@@ -58,6 +58,19 @@ impl LocalDocker {
         }
     }
 
+    /// Read Dockerfile content from location path and return an array of bytes
+    fn get_dockerfile_content(&self, dockerfile_path: &str) -> Result<Vec<u8>, EngineError> {
+        match fs::read(dockerfile_path) {
+            Ok(bytes) => Ok(bytes),
+            Err(err) => {
+                let error_msg = format!("Can't read Dockerfile '{}'", dockerfile_path);
+                error!("{:?}", error_msg);
+                error!("{:?}", err);
```

Comment:
Better to combine those two making sure log is not far, very handy when a lot of stuff is running concurently:
```
error!("Can't read Dockerfile '{}', error: {:?}", dockerfile_path, err);
```

Reply:
thank you